src/api/todos.ts

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+const USER_ID = 11528;
```

Comment:
Move it to the constants folder/file and use it, as you create the same constant in the App.tsx

Reply:
Not fixed 😢